.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgresql://postgres@db/test_track_development
```

Comment:
We could, but then if other users wanted to change it for themselves they would have to fork the whole repo